R/rtables_utils.R

```diff
@@ -390,21 +368,6 @@ count_children <- function(x) {
   ))
 }
 
-#' @export
-#' @rdname report_null
-null_report <- rtables::rtable(
-  header = "",
-  rrow("", "Null Report: No observations met the reporting criteria for inclusion in this output.")
-)
-
-#' @export
-#' @rdname report_null
-null_listing <- rlistings::as_listing(
-  df = data.frame(x = formatters::with_label(
-    "Null Report: No observations met the reporting criteria for inclusion in this output.", ""
-  ))
-)
```

Comment:
Now these are the same right? because there is no real need to differentiate, correct?

Reply:
This was not used, so I removed it from the package. I think it is easier if we know that null report will always be of class Elementary table rather than sometimes also df_listing

Reply:
It makes sense considering that is much more covered by tests and so on. Also thinking at recent bug fixes this was not considered a lot